src/alto-ui/Datagrid/Datagrid.js

```diff
@@ -117,12 +120,17 @@ class Datagrid extends React.PureComponent {
       }
       this.setState({ loaded: true });
     }, 0);
+    this.handleWindowResize();
```

Comment:
we usually use `handle...` for event listener for example `handleClickOnMyButton`. Here we just call a method that will add an event listener on window, so perhaps `this.addWindowResizeListener`. Or you can also put directly the code here (no particular gain for extracting this into a separated method)